Sources/Wrap.swift

```diff
@@ -261,7 +261,11 @@ public extension WrapCustomizable {
 public extension WrapCustomizable {
     /// Convert a given property name (assumed to be camelCased) to snake_case
     func convertPropertyNameToSnakeCase(propertyName: String) -> String {
-        let regex = try! NSRegularExpression(pattern: "(?<=[a-z])([A-Z])|([A-Z])(?=[a-z])", options: [])
+        #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
+            let regex = try! NSRegularExpression(pattern: "(?<=[a-z])([A-Z])|([A-Z])(?=[a-z])", options: [])
```

Comment:
How about adding a typealias instead, to avoid having conditional compilation inline? This could be added at the top of the file:

```swift
#if !os(Linux)
typealias RegularExpression = NSRegularExpression
#endif
```